src/applications/mhv-medical-records/util/helpers.js

```diff
@@ -353,15 +363,25 @@ export const formatDate = str => {
  *
  * @param {Date} date
  */
-export const formatDateAndTime = date => {
-  const hours = date.getHours();
-  const minutes = date.getMinutes();
+export const formatDateAndTime = rawDate => {
+  let date = rawDate;
+  if (typeof rawDate === 'string') {
+    date = new Date(rawDate);
+  }
+
+  const hours = date.getUTCHours();
+  const minutes = date.getUTCMinutes();
   const period = hours >= 12 ? 'p.m.' : 'a.m.';
   const formattedHours = hours % 12 || 12; // Convert to 12-hour format
   const formattedMinutes = minutes < 10 ? `0${minutes}` : minutes;
   const timePart = `${formattedHours}:${formattedMinutes} ${period} ET`;
 
-  const options = { year: 'numeric', month: 'long', day: 'numeric' };
+  const options = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    timeZone: 'UTC',
+  };
   const datePart = date.toLocaleDateString('en-US', options);
```

Comment:
It looks like this will return the date in UTC, but this one needs to be in the user's local time zone.